src/main/webapp/app/course/manage/course-group.component.html

```diff
@@ -102,9 +106,7 @@ <h2>
                                 deleteQuestion="artemisApp.course.courseGroup.removeFromGroup.modalQuestion"
                                 (delete)="removeFromGroup(value)"
                                 [dialogError]="dialogError$"
-                            >
-                                <fa-icon [icon]="faUserSlash" class="me-1"></fa-icon>
-                            </button>
+                            ></button>
```

Comment:
Why do you remove the icon of the button here?

Reply:
No, you are removing the icon from a different unrelated button instead of just adding a new button. I think the removal here should be reverted and the Typescript file should contain both icons, as the two icons are needed for different buttons.

Reply:
You right! 
I was deleting the icon of the remove button. 
Thank you for your observation. I reverted that.